src/circle.rs

```diff
@@ -1,9 +1,12 @@
 //! Implementation of circle shape.
 
 use std::f64::consts::{FRAC_PI_2, PI};
-use std::ops::{Add, Sub};
+use std::{
```

Comment:
if we're moving to this notation I would also combine the other `std` import, above.

Reply:
I don't mind at all which to use - should I use the separate-line way to be more consistent?